widgetssdk/src/main/java/com/glia/widgets/core/secureconversations/domain/GetUnreadMessagesCountWithTimeoutUseCase.kt

```diff
@@ -18,14 +18,16 @@ private const val TAG = "GetUnreadMessagesCountUseCase"
 @VisibleForTesting
 const val TIMEOUT_SEC = 3L
 
+internal const val NO_UNREAD_MESSAGES = -1
```

Comment:
It should look more logical to me if the value of `NO_UNREAD_MESSAGES` was `0` instead of `-1`. Is it possible?

Reply:
fixed [d9a95cf](https://github.com/salemove/android-sdk-widgets/pull/578/commits/d9a95cf17ddc59bf167997cda61337a5ead43dd6)